### PR TITLE
Create .taskcluster.yml

### PR DESCRIPTION
**Pull request by owlishDeveloper**
Configuration file ireneOwl on IRC.

### CODE REVIEW

**Comment by owlishDeveloper** [comment]
@imbstack please review!

**Comment by owlishDeveloper** [comment]
@owlishDeveloper

File: .taskcluster.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    github:
+      env: true
+      events:
+        - pull_request.opened
+        - pull_request.synchronize
+        - pull_request.reopened
+        - push
+  payload:
+    maxRunTime: 3600
+    image: "quay.io/mrrrgn/ubuntu-ci:0.0.1"
```
Comment (imbstack) [inline .taskcluster.yml]: Unfortunately this line won't be very good anymore (I need to update the docs to reflect this!).

File: .taskcluster.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    command:
```
Comment (imbstack) [inline .taskcluster.yml:22]: And similarly this command will need to be updated.

This command and the `image` associated with it for the above comment is best taken from https://github.com/taskcluster/taskcluster-github/blob/master/.taskcluster.yml#L24-L28

This is just due to bit-rot on the part of the `quay.io/mrrrgn` image. The fact that the command in this version is so much longer is annoying, but perhaps we can figure out a way to fix it soon!

**Review by imbstack — CHANGES_REQUESTED** [review]
This is looking very nice! Thanks for the PR.

File: .taskcluster.yml (hunk 1 above)
Comment (owlishDeveloper) [reply .taskcluster.yml]: done in new commit!

File: .taskcluster.yml (hunk 2 above)
Comment (owlishDeveloper) [reply .taskcluster.yml:22]: Done as well!

**Review by imbstack — APPROVED** [review]

**Comment by imbstack** [comment]
This looks great! r+

I'll merge now.

**Pull request merged**